src/generators/legacy-html/utils/buildContent.mjs

```diff
@@ -76,13 +77,38 @@ const buildHtmlTypeLink = node => {
   );
 };
 
+/**
+ * Creates a history table row.
+ *
+ * @param {ApiDocMetadataChange} change
+ * @param {import('unified').Processor} remark
+ */
+const createHistoryTableRow = (
+  { version: changeVersions, description },
+  remark
+) => {
+  const descriptionNode = remark.parse(description);
+
+  // Convert node to hast to make it compatible with createElement
+  const descriptionHast = toHast(descriptionNode);
```

Comment:
If we leave it as Mdast, won't it get converted to Hast in a later stage, or has that already occured?